core/serialize_deserialize.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
I believe we can put contents of this file into `bindings.rs` there's not that much code and we try to be frugal with files.

Reply:
I just pushed a simplified version.